_overviews/scala3-macros/tutorial/macros.md

```diff
@@ -190,23 +208,24 @@ Varargs in Scala are represented with `Seq`, hence when we write a macro with a
 It is possible to recover each individual argument (of type `Expr[T]`) using the `scala.quoted.Varargs` extractor.
 
 ```scala
-import scala.quoted.Varargs
+import scala.quoted.{Varargs, *}
```

Comment:
Is this different from that?

```suggestion
import scala.quoted.*
```

Reply:
they are equivalent, I only put it to make it clear Varargs was imported, but I can go back to just the `*`

Reply:
No, I like clarity. Would it be possible to avoid the wildcard imports in this page? Or is it impractical to work with the API without a wildcard import?